tests/jobs/conftest.py

```diff
@@ -6,17 +6,9 @@
 
 
 @pytest.fixture()
-def setup_ert(tmpdir, test_data_root):
-    cwd = os.getcwd()
-    tmpdir.chdir()
-    test_data_dir = os.path.join(test_data_root, "snake_oil")
-    shutil.copytree(test_data_dir, "test_data")
-    os.chdir(os.path.join("test_data"))
-
+def setup_ert(copy_snake_oil_case_storage):  # pylint: disable=unused-argument
```

Comment:
setup_ert sounds like it should at least return ert and not a libresfacade :shrug:

Reply:
True, did not change the name though, but added another commit to do that